common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -44,6 +49,9 @@
   public static final int DOUBLE_ARRAY_OFFSET;
 
   private static final boolean unaligned;
+
+  private static final MemoryBlock _doubleBuffer;
```

Comment:
This is a globally shared variable -- that's not going to work is it? reads would all corrupt each other.